operators/operator_sdk/osdk-scorecard.adoc

```diff
@@ -0,0 +1,16 @@
+[id="osdk-scorecard"]
+= Validating Operators using the scorecard
+include::modules/common-attributes.adoc[]
+:context: osdk-scorecard
+
+toc::[]
+
+Operator authors should validate that their Operator is packaged correctly and
+free of syntax errors. As an Operator author, you can use the Operator SDK's
+scorecard tool to validate your Operator packaging and run tests.
+
+include::modules/osdk-about-scorecard-tool.adoc[leveloffset=+1]
+include::modules/osdk-scorecard-config.adoc[leveloffset=+1]
+include::modules/osdk-running-scorecard.adoc[leveloffset=+1]
+include::modules/osdk-scorecard-tests.adoc[leveloffset=+1]
+include::modules/osdk-scorecard-olm.adoc[leveloffset=+1]
```

Comment:
Just wondering if the "Running the scorecard with an OLM-managed Operator" sections should be next to the "Running the scorecard" section? Or does it makes sense to have the tests section in between them?

Reply:
I'd thought about putting "tests" before "running". I will do this thing.